Makefile

```diff
@@ -54,7 +54,6 @@ build_tags_comma_sep := $(subst $(empty),$(comma),$(build_tags))
 
 ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=wasm \
 		  -X github.com/cosmos/cosmos-sdk/version.AppName=wasmd \
-		  -X github.com/CosmWasm/wasmd/cmd/wasmcli/version.ClientName=wasmcli \
```

Comment:
As mentioned in the comment, maybe we don't remove the CLI stuff, just keep it for Windows?